dev_guide/builds.adoc

```diff
@@ -341,19 +345,12 @@ This ensures that the triggered build uses the new image that was just pushed to
 the repository, and the build can be re-run anytime with exactly the same
 inputs.
 
-For link:../architecture/core_objects/builds.html#sti-build[STI type builds],
-the field that is matched and replaced is the `*stiStrategy.image*` field. For
-Docker builds, the field is `*dockerStrategy.baseImage*`. For custom builds, the
-`*customStrategy.image*` field is updated.
-
-In addition, for custom builds, the `*OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE*`
-environment variable is checked. If it does not exist, then it is created with
-the immutable image reference. If it does exist and the value matches the image
-field of the image change trigger, then it is updated with the immutable image
-reference.
+In addition to setting the image field for all `*Strategy*` types, for custom builds, 
+the `OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE` environment variable is checked. If it does 
+not exist, then it is created with the immutable image reference. If it does exist 
+then it is updated with the immutable image reference.
 
-If an `*imageChange*` trigger is defined on a `*BuildConfig*` and a build is
-triggered (due to an image change, webhook trigger, or manual request), then the
+If build is triggered due to a webhook trigger or manual request,
 build that is created uses the `*immutableid*` resolved from the
-`*ImageRepository*` pointed to by the `*imageChange*` trigger. This ensures that
+`*ImageStream*` referenced by the `*Strategy*`.  This ensures that builds
 builds are performed using consistent image tags for ease of reproduction.
```

Comment:
There's a couple missing "a"s and a "builds builds" in this paragraph; I could get this in a follow-up, either way.